docs/modules/ROOT/pages/how-tos/workflows/keep-remote-pipelines-up-to-date.adoc

```diff
@@ -0,0 +1,107 @@
+= Maintaining remote Tekton pipelines up to date
+
+[IMPORTANT]
+====
+* This article took for granted the reader knows what a link:https://tekton.dev/docs/pipelines/resolution/[Tekton remote pipeline] is. If this is not the case, please read more about this feature before proceeding.
+====
+
+== The advantage of using a remote pipeline
+
+This approach allows a team or an individual contributor to manage plenty of Pipelines from a single repository as source of truth. But there also other good reasons:
+
+. Pipelines are stored in a Git repository and fetched during execution, eliminating the need to maintain local or static pipeline definitions.
+. Teams can leverage Git’s versioning to access specific pipeline branches
+. In case of issues, it's very simple to rollback to a previous pipeline version, and apply it globally among the Konflux Components.
+. Reduced maintenance overhead
+
+To start referencing a remote Pipeline from a PipelineRun using a *git resolver*, replace *pipelineSpec* with *pipelineRef*:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata: ... # Omitted for brevity
+spec:
+  params: ... # Omitted for brevity
+  pipelineRef:
+    resolver: git
+    params:
+      - name: org
+        value: redhat
+      - name: scmType
+        value: gitlab
+      - name: serverURL
+        value: https://gitlab.cee.redhat.com
+      - name: url
+        value: "https://gitlab.cee.redhat.com/your-group-name/your-repo-name.git"
+      - name: revision
+        value: main # Specify a branch to target
```

Comment:
It is possible to reference a specific revision as well. I guess the benefit of using a "floating" reference like this is that the rest of the components get updates without having to get a nudge. They will get the updates in their next commit.